src/lib/extractor/extract.ts

```diff
@@ -101,6 +95,15 @@ export function fromProject(opts: Options): Doc.DocPackage {
     prjDir = process.cwd()
     debug('prjDir set to %s -- taken from current working directory', prjDir)
   }
+  const project =
+    opts.project ??
+    new tsm.Project({
+      tsConfigFilePath: path.join(prjDir, 'tsconfig.json'),
+      compilerOptions: {
+        skipLibCheck: true,
+        skipDefaultLibCheck: true
```

Comment:
Can you update the README docs about what's going on here?

Reply:
This just slipped in. Will remove it